### PR TITLE
feat: deploy on sepolia

### PR DESCRIPTION
**Pull request by alextnetto**
- Deployed new treasury
- updated ENS contract addresses on Sepolia in `constant.ts`
- updated hardhat configs 
- Deployed Namekit contract on Sepolia
- updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +60,7 @@ sequenceDiagram
         -  If you want a Safe Multisig, then create your treasury [here](https://smold.app/safe).
     2. The deployer private key must be a fresh account, with zero transactions. Make sure to send some ETH to this account in order to pay for fees on contract deployment.
 4. Deploying contracts
-   - Goerli
+   - sepolia
```
Comment (lightwalker-eth) [inline README.md:63]:
```suggestion
   - Sepolia
```

File: README.md [hunk 2]
```diff
@@ -74,7 +75,7 @@ sequenceDiagram
 ## Commands
 Before running these commands, make sure to:
 - Generate the typechain interfaces: `yarn compile`
-- Change CHAIN_ID in `.env` to the desired network you want to interact with. 1 = Mainnet, 5 = Goerli.
+- Change CHAIN_ID in `.env` to the desired network you want to interact with. 1 = Mainnet, 11155111 = sepolia.
```
Comment (lightwalker-eth) [inline README.md:78]:
```suggestion
- Change CHAIN_ID in `.env` to the desired network you want to interact with. 1 = Mainnet, 11155111 = Sepolia.
```

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "compile": "npx hardhat compile",
-    "deploy:test": "npx hardhat run script/deploy/deploy.ts --network goerli",
+    "deploy:test": "npx hardhat run script/deploy/deploy.ts --network sepolia",
```
Comment (lightwalker-eth) [inline package.json:12]:
```suggestion
    "deploy:sepolia": "npx hardhat run script/deploy/deploy.ts --network sepolia",
```

File: README.md [hunk 4]
```diff
@@ -59,7 +60,7 @@ sequenceDiagram
         -  If you want a Safe Multisig, then create your treasury [here](https://smold.app/safe).
     2. The deployer private key must be a fresh account, with zero transactions. Make sure to send some ETH to this account in order to pay for fees on contract deployment.
 4. Deploying contracts
-   - Goerli
+   - sepolia
    ```
    yarn deploy:test
```
Comment (lightwalker-eth) [inline README.md:65]:
```suggestion
   yarn deploy:sepolia
```

What do you think about updating this to be more specific about the testnet? Would prefer to see a specific reference to sepolia here.

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@alextnetto Hey thanks for the updates. Reviewed and shared feedback :+1:

**Review by alextnetto — COMMENTED** [review]
- Deployed new treasury
- updated ENS contract addresses on Sepolia in constant.ts
- updated hardhat configs
- Deployed Namekit contract on Sepolia
- updated README

**Review by lightwalker-eth — APPROVED** [review]
@alextnetto Updates look nice. Thanks! Approved :+1:

**Pull request merged**